docs/attribute.md

```diff
@@ -0,0 +1,37 @@
+Annotation
```

Comment:
Attributes

Reply:
Sorry, I quickly copied the annotations documentation. It's corrected.